package.json

```diff
@@ -36,7 +36,7 @@
     "babel-polyfill": "^6.26.0",
     "babel-preset-env": "^1.6.1",
     "chromedriver": "85.0.0",
-    "fast-csv": "^2.4.1",
+    "fast-csv": ">=4.3.6",
```

Comment:
Based on the definition of "^", I prefer to use "^" instead of ">=". @BruceDai

Reply:
@ibelem  Thanks for your suggestion, I've updated the PR, please take another look, thanks.